### ISSUE TITLE
[RFC] Multi-language support in mis_builder

### ISSUE THREAD

**Issue by pedrobaeza** [OCA/mis-builder#140]
For having this, I'm thinking in:

* Make KPI's description field translatable.
* Language selection as mis.report.instance level.
* Pass language context on values generation.
* Pass language context on report/XLS generation.

@sbidoul anything more I'm not counting?

**Comment by sbidoul** [comment]
So you want to print a report in a different language than the user language?

Your list sounds plausible, I've not investigated in detail. In any case if the context is not propagated somewhere it's considered a bug.

**Comment by pedrobaeza** [comment]
Yes, the idea is to show and print the mis builder template in other specific language. Example: a foreign bank asks you to give them a balance sheet, so you want to print Spanish Balance Sheet, but in English.

I'm quoting this right now, so I will tell you if this moves forward.

**Comment by HaraldPanten** [comment]
Hi @pedrobaeza , @sbidoul 

KPI Description field is translatable for V11, already. It's still not translatable in V10. (I don't know in previous versions). I'm not sure if it's an improvement for V11 MIS-Builder or if it's some kind of "basic" change/improvement for base Odoo V11.

Few days ago we had the same issue. We needed to print the report in English from a Spanish Odoo.

Our "povisional" solution was to search the KPI descriptions in translations settings. After that, we translated the original term to English, keeping the translation in Spanish.

Now we can change users language preferences and print the report in Spanish or English, depending on our needs.

Even though, I think that this new suggested enhancement could be really useful.
